octobot_evaluators/evaluator/abstract_evaluator.py

```diff
@@ -30,19 +30,22 @@ class AbstractEvaluator(AbstractTentacle):
 
     def __init__(self):
         super().__init__()
+        # Global OctoBot configuration
         self.config = None
+
+        # Evaluator specific config (Is loaded from tentacle specific file)
         self.specific_config = {}
 
         # If this indicator is enabled
         self.enabled = True
 
-        # Symbol is the cryptocurrency symbol
+        # Symbol is the cryptocurrency pair (Should be None if wildcard)
         self.symbol = None
```

Comment:
For not-pair related evalautors (ex: socials) this might be the name of the currency, comment should be something like:
```# Symbol is the cryptocurrency pair (Should be None if wildcard) or the name of the currency if evaluator is not trading pair related```

Reply:
Yes good idea :+1:

Reply:
Or maybe for cryptocurrency related evaluator a new attribute ? Like `self.cryptocurrency` ?

Reply:
Right, we can also do that and I think it would be more correct :)

Reply:
Done in https://github.com/Drakkar-Software/OctoBot-Evaluators/pull/13